tests/expressions/tests.py

```diff
@@ -981,7 +981,7 @@ def test_deepcopy(self):
     def test_deconstruct(self):
         f = F('name')
         path, args, kwargs = f.deconstruct()
-        self.assertEqual(path, 'django.db.models.expressions.F')
```

Comment:
I'm not sure about doing this only for `F()`, why `F()` deserves special treatment?

Reply:
I just wrote a check constraint using `Q()` and `F()` and was surprised that `Q()` already has this treatment whilst `F()` does not.

Reply:
But yes there are probably more classes that could have such treatment.

Reply:
`Q()` is not a real "expression" and it deserves special treatment, see ticket-29125. IMO it makes sense to do this change only for all expressions (as for `django.db.models.fields.*`), but not all of them are importable from `django.db.models` so it's more complicated.

Reply:
I agree it makes sense. Can we defer that to a ticket and merge this change now? This PR can then serve as an example to someone who'd want to pick up changing all other expressions.